scripts/utility.py

```diff
@@ -278,6 +278,8 @@ def change_clan_reputation(difference):
     will change the Clan's reputation with outsider cats according to the difference parameter.
     """
     game.clan.reputation += difference
```

Comment:
This is also an option
`game.clan.reputation = clamp(game.clan.reputation + difference, 0 , 100) `